src/pages/Home/Home.js

```diff
@@ -10,8 +10,8 @@ import './_pillar.home.source.scss';
 const Home = () => (
   <div>
     <Header img={Images.header} startColor="#667EEA" endColor="#764BA2">
-      <h1>Forge The Future</h1>
-      <h2>BoilerMake | Purdue University </h2>
+      <h1>Forge the Future</h1>
```

Comment:
I think we can take out "forge the future" since that was our tagline last year. We could do
BoilerMake VII
Purdue University
January 24-26, 2020

Reply:
3 lines didn't look great, went with this instead